### DISCUSSION TITLE
"Read More" Links

### DISCUSSION THREAD

**Discussion by shkm** [jpanther/congo#275] [Q&A]
I like to use the ` <!--more-->` annotation with my blog posts, to break them up where I see fit. I'd love to have a "Read more" link in place of it or something; right now it just renders nothing and can make it look like the post just ends there.

Any chance of this? Can we configure it now?

**Comment by jpanther** [comment]
Thanks for the suggestion. Are you referring to having "Read more" appear underneath the article summary on the list pages? I'm struggling to find any Hugo documentation on how `<!--more-->` actually works.
